package.json

```diff
@@ -24,6 +24,8 @@
     "webpack": "^1.13.0"
   },
   "devDependencies": {
+    "babel-core": "^6.9.1",
+    "babel-loader": "^6.2.4",
```

Comment:
TBH though I'm not yet familiar with the finer distinctions between `dependencies` and `devDependencies`. Since we're just serving static files in production and not running node, maybe everything should end up in `devDependencies`? Not suggesting this get tackled here. Just thinking 'aloud.'